README.md

```diff
@@ -21,87 +21,77 @@
   </a>
 </p>
 
-<h1 align="center" >The Context Layer for your Metrics</h1>
-<p align="center">Supercharge your semantic layer with context for your data consumers.
-Build actionnable and trusted metrics with changelog, monitoring and centralized interpretation.
+<h1 align="center" >Data versioning and diffing</h1>
+
+<p align="center"> Datadrift is an agnostic and lightweight storage and version-control technology to track changes to mutable data sources
 </p>
 
-<p align="center"><a href="https://data-drift.io">Website</a> · <a href="https://www.data-drift.io/blog">Blog</a> · <a href="https://github.com/data-drift/data-drift/issues">Issues</p>
-</br>
+<p align="center"><a href="https://data-drift.io">Website</a> · <a href="https://www.data-drift.io/blog">Blog</a> · <a href="https://github.com/data-drift/data-drift/issues">Issues
+</p>
 
 <p align="center">
   <a href="https://www.data-drift.io">
-    <img src="./datadrift-new-drift.png" alt="DataDrift hero with metric volatility charts" />
+    <img src="./datadrift-storage-diff-schema.png" alt="DataDrift " />
   </a>
 </p>
-</br>
+
 
 # 👋 About
 
-## The Problem: Data consumers interact with raw metrics only. Context is key to make metrics trusted and actionable.
+## 🥵 Storing and handling data history is complex and expensive
+Tools, databases and warehouses have a hard time tracking and displaying historical changes.
+- For a very large majority of companies, there no access to historical state of their own data (ie. how data changes over time)
 
-</br>
-Most data consumers never query a data warehouse table, yet use data on a daily basis through the lens of metrics.
+- For a selected few, keeping track of historical changes is made at a great cost of data engineering and outdated modeling trade-offs
 
-To trust metrics and make decisions based on them, we need to guarante the quality of the metrics itself (not only the underlying tables) and give (a lot of) context around it.
+## 👉 Open-source versioned storage and dedicated tools to work with data history
+DataDrift makes handling data history easy with modern and open-source version control tools for data. 
 
-</br>
-Context is vital because is gives data consumers awareness of:
+### Simple & ligthweight techno for all
+- **Easy to implement (<15min)**: Historize changes in 1 click. Intuitive to use for both technical and non-technical users
 
-- **Computation**: how a metric was calculated and associated caveats
+- **Cost efficient**: Reduce your storage and optimize your warehouse bill with our ligthweight storage for data history
```

Comment:
- **Free**: Storage is done in a dedicated git repository, no additional cost if you use Github